samples/config/dev/benchmark-am/amster.yaml

```diff
@@ -0,0 +1,13 @@
+config:
+  claim: frconfig
+  importPath: /git/config/6.5/smoke-tests/am/
+  exportPath: /tmp/amster
+
+version: 6.0.0
```

Comment:
Should this remain 6.0.0?

Reply:
I am just moving things around so it should not effect this PR